pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[build-system]
```

Comment:
You probably want to combine this with an action that does a "empty python env with nothing but pip" install. That way you know it's working fine.